megamek/src/megamek/common/LandAirMech.java

```diff
@@ -2108,4 +2115,82 @@ public void updateSensorOptions() {
             }
         }
     }
+    
+    /**
+     * @return Returns the autoEject.
+     */
+    public boolean isAutoEject() {
+        if (game.getOptions().booleanOption(OptionsConstants.ADVANCED_STRATOPS_QUIRKS)) {
+            return !hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT);
+        }
+        return true;
```

Comment:
As with Aero, this ignores the player's configuration and forces autoeject to be active unless otherwise dictated by the quirk. The check for the quirk should probably be added to Mech#isAutoEject.

Reply:
Fixed